pkg/ring/lifecycler.go

```diff
@@ -673,7 +723,7 @@ func (i *Lifecycler) initRing(ctx context.Context) error {
 		// else set to PENDING
 		if instanceDesc.State == LEAVING && len(instanceDesc.Tokens) != 0 {
```

Comment:
For L722, do we need to update the comment? Since we are not doing `move it into ACTIVE` anymore but based on its previous state